assets/prefabs/potion/DivergencePotion.prefab

```diff
@@ -0,0 +1,33 @@
+{
+    "parent": "engine:iconItem",
+    "Item": {
+        "icon": "Potions:DivergencePotionBottle",
+        "stackId": "Potions:divergencePotion",
+        "consumedOnUse": true
+    },
+    "DisplayName": {
+        "name": "Potion of Divergence",
+        "description": "Coincidentally heals and damages the player for 10 seconds, but heals in larger amounts."
+    },
+    "Potion": {
+        "hasGenome": false,
+        "effects": [
+            {
+                "effect": "HARM",
+                "magnitude": 3,
+                "duration": 10100
+            }, 
+            {
+                "effect": "HEAL",
```

Comment:
Rather than HEAL maybe make it REGEN? As configured you don't see the effect since both of them are instant. So you could lose a set amount of HP up front but then regen more than that over a short duration. Check out one of the regen potions to see how they work :-)